feconf.py

```diff
@@ -520,6 +520,12 @@ def get_empty_ratings():
 UPDATE_TYPE_COLLECTION_COMMIT = 'collection_commit'
 UPDATE_TYPE_FEEDBACK_MESSAGE = 'feedback_thread'
 
+# User query status.
```

Comment:
I don't understand what do you mean here ?
Keeping documentation for "state diagram" . I don't understand what do you mean by "state diagram".